tests/v2/validation/provisioning/rke1/provisioning_node_driver_test.go

```diff
@@ -76,6 +76,10 @@ func (r *RKE1NodeDriverProvisioningTestSuite) TestProvisioningRKE1Cluster() {
 	nodeRolesAll := []provisioninginput.NodePools{provisioninginput.AllRolesNodePool}
 	nodeRolesShared := []provisioninginput.NodePools{provisioninginput.EtcdControlPlaneNodePool, provisioninginput.WorkerNodePool}
 	nodeRolesDedicated := []provisioninginput.NodePools{provisioninginput.EtcdNodePool, provisioninginput.ControlPlaneNodePool, provisioninginput.WorkerNodePool}
+	nodeRolesDedicated[0].NodeRoles.Quantity = 3
```

Comment:
Why are we changing it to this? The static tests are testing the various cluster configurations. If anything, we should probably add 3, 2, 3 as a static option, but not the only option.